src/com/palmergames/bukkit/towny/Towny.java

```diff
@@ -442,9 +442,11 @@ private void update() {
 		return townyUniverse;
 	}
 
-	public String getVersion() {
-
-		return version;
+	public Version getVersion() {
+		if (CUR_PLUGIN_VER == null) {
+			CUR_PLUGIN_VER = BukkitTools.getTownyVersion();
```

Comment:
Because the cur_plugin_ver is being set using the bukkittools.getTownyVersion() it will always show the stripped out version, dropping any of the ` for 1.12.2`.

Either this line should be `CUR_PLUGIN_VER = this.getDescription().getVersion();` and the Version is parsed through BukkitTools.getTownyVersion() later on in TownySettings.isTownyUpdating(), or the version variable removed from the above needs to be restored and used in the safemode/enabled/disabled messages.

Doing the former (parsing getTownyVersion() in TownySettings.isTownyUpdating()) would mean that passing the version number through is unnecessary(ie `TownySettings.isTownyUpdating(getVersion())`), as the version is always available via BukkitTools.